### ISSUE TITLE
Brain_Data iplot and plot produce different percentile thresholding

### ISSUE THREAD

**Issue by ejolly** [cosanlab/nltools#311]
Several things to note and they might be related to https://github.com/cosanlab/nltools/issues/310:

1) The way `iplot` works is that it calls `nilearn.plotting.view_image` under the hood which computes its threshold using [this helper function](https://github.com/nilearn/nilearn/blob/ded4df9f6f1cf5c44a0f56d01fd1e951a079db9b/nilearn/_utils/extmath.py#L10). It turns out this is computed over _the entire nifti image_ with no ability specify a mask. This means the percentile threshold will routinely be _lower_ for`brain.iplot()` than `brain.plot()` producing different figures. See the issue I opened here to track this is on the nilearn side: https://github.com/nilearn/nilearn/issues/2112

2) `iplot` also has no way of specifying something comparable to `threshold_lower` in `plot` and instead uses the absolute value of the image and a single threshold. This means that maps that have negative and positive values will have different percentile bounds with both methods. `plot` takes into account the fact that distributions can be non-symmetrial whereas `iplot` does not

3) Alsop worth noting that when percentile thresholding is performed on maps constructed using ROI masks, the bounds are determined using all _voxels_ rather than _rois_, which means that the bounds are actually a bit _more stringent_ than they ought to be. 

TL;DR **Dont use percentile thresholding with `Brain_Data.iplot()` for the moment. It produces results different from what you probably want**.

**Comment by ejolly** [comment]
Resolved: The `iplot()` method has been removed in v0.6.0. Users should use `plot()` exclusively, which handles percentile thresholding correctly within the mask. The underlying issue with nilearn's `view_img` computing thresholds over the entire nifti is no longer relevant since we don't wrap it in `iplot()` anymore.
